README.md

```diff
@@ -2,6 +2,42 @@
 
 Bring [dapr](https://dapr.io/) to Haskell world.
 
+## Local development
+
+### Install the Dapr CLI
+
+See [Install the Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+
+### Clone code
+
+```bash
+git clone https://github.com/nodew/haskell-dapr.git
+
+cd haskell-dapr
+```
+
+### Run tests
+
+#### Setup dependencies
+
+```bash
+./scripts/test-init.sh
+```
+
+#### Init dapr
+
+```bash
+dapr init
+
+dapr run --app-id haskell-dapr --dapr-http-port 3500 --components-path ./test/components
```

Comment:
We can also run the test here. Example, https://github.com/dapr/js-sdk/blob/fd273a9adefa7c328b9452052d8d8008fca91021/package.json#L16

This should work
`dapr run --app-id haskell-dapr --dapr-http-port 3500 --components-path ./test/components -- stack test`